src/main/webapp/scripts/common.js

```diff
@@ -15,11 +15,11 @@
 const scrollTopButton = document.getElementById("to-top-button");
```

Comment:
can you please move this inside the `onscroll` function? There's no need need to have it in the global scope

Reply:
Thanks!